packages/adapters/transports/webhooks/src/index.test.ts

```
@@ -0,0 +1,82 @@
+import { afterEach, describe, expect, it, vi } from "vitest";
+import { WebhookTransport } from "./index";
+
+export {};
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)